CS50/Week_2/Problem_set/Problem_set/wordle50/wordle50.c

```diff
@@ -0,0 +1,80 @@
+// The idea of this task is to re-create famous web-game 'Wordle50'.
+// Idea of the game is to guess the 5-letter code word - in 6 guesses, in which
+// the player has to guess - when player uses charcter that is in code word,
+// it is marked as green (if it's in correct place) or yellow (if its in wrong
+// place). Otherwise the character is marked as red.
+
+#include <ctype.h>
+#include <stdio.h>
+#include <string.h>
+#include <strings.h>
+#include "C:\\Users\\Luksi02\\Desktop\\CS50\\Week_2\\cs50.h"
+
+string get_answer(void);
+
+int main(void)
+{
+    string codeworld = "world"; // simple example
+
+    printf("You have 6 tries to guess 5-letter codeword, good luck! \n");
+
+    for (int i = 1; i < (strlen(codeworld) + 2); i++)
+    {
+        string answer = get_answer();
+        int win = strcmp(codeworld, answer);
+        if (win == 0) // if words are equal => 0
+        {
+            return printf("Guess no. %i You guessed the codeword! Congratulations!", i);
+        }
+        else
+        {
+            for (int j = 0; j < strlen(codeworld); j++)

```

Comment:
hmm what computational complexity will this have? ?
n squared
we could do a bit better, think if it's possible to do this without two for loops ;)